mlir/test/lib/Dialect/Vector/TestVectorTransforms.cpp

```diff
@@ -776,6 +776,32 @@ struct TestFoldArithExtensionIntoVectorContractPatterns
     (void)applyPatternsAndFoldGreedily(getOperation(), std::move(patterns));
   }
 };
+
+struct TestVectorMaskedLoadStoreLowering
+    : public PassWrapper<TestVectorMaskedLoadStoreLowering,
```

Comment:
nit: we can make the struct final and use `override` uniformly across methods
```suggestion
struct TestVectorMaskedLoadStoreLowering final
    : PassWrapper<TestVectorMaskedLoadStoreLowering,
```